tests/strands/tools/executors/conftest.py

```diff
@@ -0,0 +1,117 @@
+import threading
+import unittest.mock
+
+import pytest
+
+from strands import tool as sa_tool
+from strands.experimental.hooks import AfterToolInvocationEvent as SAAfterToolInvocationEvent
+from strands.experimental.hooks import BeforeToolInvocationEvent as SABeforeToolInvocationEvent
```

Comment:
Personally, the SA prefixes aren't adding anything except noise.  Every time I see it I wonder what it means

What's the purpose of them?

Reply:
If there's an import conflict (or we're alias a common term like `tool`) then I think an alias makes sense.  In this cases I'm opposed to it for two reasons:

 - It's inconsistent; all other imports are not prefixed and throughout the code-base we're not really using aliases
 - the SA prefix is just making the names longer with no benefit IMHO